src/cmd/cloud.js

```diff
@@ -779,11 +779,24 @@ module.exports = class CloudCommand extends CLICommandBase {
 	 */
 	_processDirIncludes(fileMapping, dirname, { followSymlinks } = {}){
 		dirname = path.resolve(dirname);
+		let files = new Set();
 
-		const includesFile = path.join(dirname, settings.dirIncludeFilename);
-		const ignoreFile = path.join(dirname, settings.dirExcludeFilename);
-		let hasIncludeFile = false;
+		this._getDefaultIncludes(files, dirname, { followSymlinks });
+		this._getCustomIncludes(files, dirname, { followSymlinks });
+		this._getDefaultIgnores(files, dirname, { followSymlinks });
+		this._getCustomIgnores(files, dirname, { followSymlinks });
 
+		// Add files to fileMapping
+		Array.from(files.values()).sort();
+		files.forEach((file) => {
```

Comment:
Interestingly this works but you're not getting the benefits of the sort
```suggestion
		const sortedFiles = Array.from(files.values()).sort();
		sortedFiles.forEach((file) => {
```